src/components/Auth.tsx

```diff
@@ -1,47 +1,56 @@
-import React, { useContext } from 'react'
-import { AppContext } from './../Context'
-import { getAuth, GoogleAuthProvider, OAuthProvider, getRedirectResult, signInWithRedirect } from 'firebase/auth'
-import { FaApple } from 'react-icons/fa'
-import { FcGoogle } from 'react-icons/fc'
-import '../styles/Auth.css'
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { getAuth, getRedirectResult, GoogleAuthProvider, OAuthProvider, signInWithPopup } from "firebase/auth";
+import { FaApple } from "react-icons/fa";
+import { FcGoogle } from "react-icons/fc";
 
-const Auth: React.FC = () => {
-  const { userState } = useContext(AppContext)
-  const [, setUser] = userState
-
-  const auth = getAuth()
-  const appleProvider = new OAuthProvider('apple.com')
-  appleProvider.addScope('email')
-  appleProvider.addScope('name')
-  const googleProvider = new GoogleAuthProvider()
-  const logIn = (auth: any, provider: any): void => {
-    void signInWithRedirect(auth, provider)
+const Auth = () => {
+  const auth = getAuth();
+  const googleProvider = new GoogleAuthProvider();
+  const appleProvider = new OAuthProvider("apple.com");
+  appleProvider.addScope("email");
+  appleProvider.addScope("name");
+  const logIn = (auth: any, provider: any) => {
+    void signInWithPopup(auth, provider)
+    localStorage.setItem("uuid", "something");
```

Comment:
What purpose does this serve? The abandoned login issue is solved if I yeet this line.